README.rst

```diff
@@ -12,31 +12,99 @@ perf
 
 The Python ``perf`` module is a toolkit to write, run and analyze benchmarks.
 
-Features:
+Features
+--------
 
 * Simple API to run reliable benchmarks
 * Automatically calibrate a benchmark for a time budget.
 * Spawn multiple worker processes.
 * Compute the mean and standard deviation.
 * Detect if a benchmark result seems unstable.
-* ``perf stats`` command to analyze the distribution of benchmark
-  results (min/max, mean, median, percentiles, etc.).
+* JSON format to store benchmark results.
+* Support multiple units: seconds, bytes and integer.
+
+Usage
+-----
+
+To `run a benchmark`_ use the ``perf timeit`` command::
+
+    $ python3 -m perf timeit '[1,2]*1000' -o bench.json
+    .....................
+    Mean +- std dev: 4.22 us +- 0.08 us
+
+or write a benchmark script:
+
+.. code:: python
+
+    #!/usr/bin/env python3
+    import perf
+    import time
+
+
+    def func():
+        time.sleep(0.001)
+
+
+    runner = perf.Runner()
+    benchmark = runner.bench_func('sleep', func)
+    benchmark.dump('sleep.json', replace=True)
```

Comment:
I'm curious why you use that in your own benchmarks. Is it annoying to have to write "-o sleep.json" on the command line? Or is it because perf fail if the file already exists?

Not replacing by default is a deliberate choice to avoid mistakes. I make mistakes all the time when I run benchmarks for hours, and it's painful to loose a file which took 2 hours to be computed.